packages/nimble-components/CONTRIBUTING.md

```diff
@@ -77,15 +77,34 @@ Before building a new component, 3 specification documents need to be created:
 
 ## Develop new components
 
-### Marking a component as in development
+### Marking a component as experimental / in-development / incubating
 
-If a component will require multiple pull requests before having a complete and stable API, it should be marked as "in-development" to indicate to clients that they shouldn't start using it yet. To do this:
+If a component is not ready for general use, it should be marked as "experimental" to indicate that status to clients. A component could be in this state if it:
+
+-   is still in development
+-   is currently application-specific and hasn't yet been generalized for broader use
+-   is missing important features like interaction design, visual design, accessibility, or framework integration
```

Comment:
Just scanning through our other lists in the doc start each list item with capitals so I'd continue that pattern here and in the following